com.github.kagami.boram.yml

```diff
@@ -0,0 +1,241 @@
+app-id: com.github.kagami.boram
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Boram
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=home
+  - --filesystem=/mnt
```

Comment:
```suggestion
```

Reply:
When settings a partition to auto-mount on Gnome Disks, it mounts them on `/mnt`. Is it not standard?